src/Features/Core/Portable/ExtractMethod/ExtractMethodResult.cs

```diff
@@ -39,23 +38,17 @@ internal abstract class ExtractMethodResult
         /// </summary>
         public ImmutableArray<AbstractFormattingRule> FormattingRules { get; }
 
-        /// <summary>
-        /// the generated method node that contains the extracted code.
-        /// </summary>
-        public SyntaxNode? MethodDeclarationNode { get; }
-
         /// <summary>
         /// The name token for the invocation node that replaces the extracted code.
         /// </summary>
-        public SyntaxToken InvocationNameToken { get; }
+        public SyntaxToken? InvocationNameToken { get; }
```

Comment:
made optional to indicate that it can fail (though i know of no cases where it does).  if users report that they did an EM that didn't result in rename happening, we can investigate.  but at least the user is not blocked like they were before.